test/integration/verify/controls/azurerm_postgresql_servers.rb

```diff
@@ -2,7 +2,7 @@
 postgresql_server_name = input('postgresql_server_name', value: nil)
 
 control 'azurerm_postgresql_servers' do
-  only_if { ENV['SQL'] }
+  only_if { !postgresql_server_name.nil? }
```

Comment:
There probably should be a corresponding update [here](https://github.com/chef-partners/inspec-azure-ci/blob/ef34e0826a90be63dbe31843b7eb9db16319df7a/Jenkinsfile#L111)
Weird that this was here when there was no logic surrounding the creation of the resources.